main.go

```diff
@@ -61,11 +65,16 @@ func main() {
 		if context.GlobalBool("debug") {
 			logrus.SetLevel(logrus.DebugLevel)
 		}
+		if path := context.GlobalString("debug-log"); path != "" {
```

Comment:
maybe we should just call the flag log because all logging output would be written to the file